src/FsAutoComplete.Core/ParseAndCheckResults.fs

```diff
@@ -481,7 +481,7 @@ type ParseAndCheckResults
       Ok(symboluse, symboluses)
 
   member __.TryGetSignatureData (pos: Position) (lineStr: LineStr) =
-    match Lexer.findLongIdents (pos.Column - 1, lineStr) with
+    match Lexer.findLongIdents (pos.Column, lineStr) with
```

Comment:
fixing the first off-by-one (https://github.com/fsharp/FsAutoComplete/pull/994/files#r948031768) surfaced _this_ off-by-one.

*sigh*